testprojects/tests/java/org/pantsbuild/testproject/buildrefactor/x/BUILD

```diff
@@ -1,6 +0,0 @@
-# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
-# Licensed under the Apache License, Version 2.0 (see LICENSE).
-
-java_library(
-    name = "X",
-)
```

Comment:
You'll want to leave this file in master as well. So remove this file deletion, and add it back in when you do the revert since this test is dependent on it

https://github.com/pantsbuild/pants/blob/master/contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/test_buildozer_integration.py